src/locales/fr/PAGES/SAMPLE.json

```
@@ -0,0 +1,21 @@
+{
+    "TITLE": "Étape 3 : Sélectionner un échantillon représentatif",
+    "INTRO": "Sélectionnez les pages Web à évaluer. Les notes que vous avez prises lors de l’exploration à l’étape précédente sont présentées ici pour vous aider à sélectionner les pages Web. Veillez à ce que les pages web sélectionnées soient représentatives du site Web cible.",
+    "RESOURCE_LINK_NAME": "Étape 3 : Sélectionner un échantillon représentatif",
+    "HD_STRUCT_SAMPLE": "Échantillon structuré",
+    "HD_ESSENT_FUNC": "Fonctionnalités essentielles (notes facultatives de l’étape précédente) ",
+    "HD_VARIETY_PAGE_TYPES": "Variété des types de pages Web (notes facultatives de l’étape précédente) ",
+    "HD_STRUCT_SAMPLE_SUB": "Pages Web de l'échantillon structuré",
```

```suggestion
    "HD_STRUCT_SAMPLE_SUB": "Pages Web de l’échantillon structuré",
```

@KodGit The current apostrophe is displayed `Pages Web de l&#39;échantillon structuré`